mirror/mirror.go

```diff
@@ -112,6 +125,15 @@ func New(path string, opts NewOpts) (*Handle, error) {
 		)
 	}
 
+	if opts.ExpectedEvidencePolicy != mtc.UnsetEvidencePolicy &&
```

Comment:
This allows a mirror with `UnsetEvidencePolicy` to consume from a MTCA with `UmbilicalEvidencePolicy`. Is that intended? I suppose it should work.

Reply:
Yes. `opts.ExpectedEvidencePolicy == UnsetEvidencePolicy` means "I don't care about the evidence policy".